src/views/EditorHome.vue

```diff
@@ -672,7 +672,7 @@ const restartAppWithMultiEngineOffMode = () => {
   store.dispatch("RESTART_APP", { isMultiEngineOffMode: true });
 };
 
-const openFaq = () => {
+const openqa = () => {
```

Comment:
```suggestion
const openQa = () => {
```
```suggestion
const openQA = () => {
```

単語を区切った方がいいような気がします。

@Hiroshiba こういう、キャメル/パスカルケース内の略語ってどうするか決まってましたっけ？

Reply:
特に決まってないです！
openQaのが良いかなと思いました！

Reply:
とりあえずコミットを押しましたが
他のところが変わってないので変更します

Reply:
変更完了